utils/ratlib.py

```diff
@@ -168,6 +171,17 @@ def try_parse_uuid(suspect: str) -> UUID:
         return result
 
 
+# duration functions
+def duration(time: datetime.timedelta) -> str:
+    """
+    Converts a datetime object into a human readable duration string.
+    """
+    if not isinstance(time, datetime.timedelta):
```

Comment:
This method appears never to be tested? Also, documentation on arguments and return value.

Reply:
Tests added in 036fd0ec